static/test/javascripts/spec/commercial/creatives/template.spec.js

```diff
@@ -46,12 +45,15 @@ define([
                 offer_button: 'yes',
                 clickMacro: '%%CLICK_URL_ESC%%'
             };
-            new Template(slot, params).create().then(function () {
-                expect(document.querySelector('.commercial', slot)).not.toBe(null);
-            });
+            new Template(slot, params).create()
+            .then(function () {
+                expect(document.querySelector('.adverts', slot)).not.toBe(null);
+            })
```

Comment:
The `.commercial` class does not exist anymore. I will update these tests later.